gapic/schema/wrappers.py

```diff
@@ -359,6 +394,15 @@ def dispatch_field(self) -> Union[MessageType, EnumType, PythonType]:
         This is what is used for `functools.singledispatch`."""
         return next(iter(self.fields.values()))
 
+    @utils.cached_property
+    def composite_types(self) -> Sequence[Field]:
```

Comment:
Can we add a test for this method? It'd be nice to check that it handles both enums and messages appropriately.